pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/IngestionDelayTracker.java

```diff
@@ -156,15 +142,15 @@ public IngestionDelayTracker(ServerMetrics serverMetrics, String tableNameWithTy
   *
   * @param currentDelay original sample delay to which we will add the age of the measure.
    */
-  private long getAgedDelay(DelayMeasure currentDelay) {
-    if (currentDelay == null) {
+  private long getAgedDelay(Long ingestionTimeMs) {
```

Comment:
Update the comments and method name accordingly, same for other places

Reply:
Comment had stale parameters but function name seems appropriate.

Reply:
OK I think I fixed the legacy comments now

Reply:
Since we no longer have the measurement time concept, I feel calling it `getIngestionDelayMs()` is more clear. Same for the javadoc.

Reply:
Oh, I think I get your point now, let me rename

Reply:
Done rename with last commit